modules/helm-creating-role-bindings-to-access-multiple-chart-repositories.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="creating-role-bindings-to-access-multiple-chart-repositories_{context}"]
+= Creating role bindings to access multiple chart repositories
+
+As a cluster administrator, you can add multiple instances of Helm chart repositories, and provide authenticated users access to charts from configured repositories by adding appropriate roles to the cluster.
+
+.Procedure
+
+* Provide regular users access to browse Helm chart repositories by creating the following role and role binding:
```

Comment:
this is already the default state in the cluster, hence there is no need to document it.